src/Ninject.Web.AspNetCore.sln

```diff
@@ -46,6 +48,12 @@ Global
 		{8E2DE3C5-5B18-4136-878D-AA671FD7709C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{8E2DE3C5-5B18-4136-878D-AA671FD7709C}.Release22|Any CPU.ActiveCfg = Release22|Any CPU
 		{8E2DE3C5-5B18-4136-878D-AA671FD7709C}.Release31|Any CPU.ActiveCfg = Release31|Any CPU
+		{7DB7F499-E11E-4B45-BDFD-1883BC8C224D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{7DB7F499-E11E-4B45-BDFD-1883BC8C224D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7DB7F499-E11E-4B45-BDFD-1883BC8C224D}.Release22|Any CPU.ActiveCfg = Release22|Any CPU
```

Comment:
I am currently using the two release configurations to build the actual release packages. Since the tests should not be part of those packages, can you please disable the build of the unit test project in the build configuration? The tests can still be run with the Debug configuration which should be more than enough.

Reply:
Yes, you're right. I disabled it.